05_service_configuration/050_caching_dns_server/052_dns_zones.md

```diff
@@ -0,0 +1,95 @@
+# DNS Zones
+Based on this this [Ubuntu Wiki](https://wiki.ubuntuusers.de/DNS-Server_Bind/#Das-Anlegen-der-Zonendateien) article, the examples are taken from there.
```

Comment:
Please see my last comment about "this". It won't get better if you double the appreance of "this" :)

Reply:
Done